xtp/src/libxtp/orbitals.cc

```diff
@@ -651,11 +659,12 @@ void Orbitals::ReadFromCpt(CheckpointReader r) {
   r(qm_energy_, "qm_energy");
   r(qm_package_, "qm_package");
   try {
-    r(pm_localized_orbitals_, "PML_MOs");
+    r(lmos_, "LMOs");
   } catch (std::runtime_error& e) {
     ;
   }
 
+  r(lmos_energies_, "LMOs_energies");
```

Comment:
should be in the try statement as well, because otherwise you are not backwards compatible